benchmark/Benchmark.re

```diff
@@ -0,0 +1,204 @@
+open Core_bench.Std;
+
+let numberOfChildren = 4;
+let totalNodesAtDepth = depth => Core.Int.(numberOfChildren ** depth);
+
+module Update = {
+  type t = {
+    depth: int,
+    index: int,
+    switchComponent: bool,
+  };
+
+  let random = (~depth, ~switchComponent) => {
+    {
+      depth,
+      index: Core.Random.int(totalNodesAtDepth(depth)),
+      switchComponent,
+    };
+  };
+
+  let remoteAction: Brisk.RemoteAction.t(t) = Brisk.RemoteAction.create();
+};
+
+module Unit = {
+  open Brisk;
+  let component = nativeComponent("B");
+
+  let createElement = (~depth as _, ~children, ()) => {
+    component((_: Hooks.empty) =>
+      {
+        make: () => (),
+        configureInstance: (~isFirstRender as _, ()) => (),
+        children: listToElement(children),
+      }
+    );
+  };
+};
+module A = {
+  open Brisk;
+  open Update;
+
+  module ComponentType = {
+    type t =
+      | A
+      | A'
+      | B
+      | B';
+
+    let toggle =
+      fun
+      | A
+      | A' => B
+      | B
+      | B' => A;
+
+    let dummyUpdate =
+      fun
+      | A => A'
+
+      | A' => A
+      | B => B'
+      | B' => B;
+  };
+
+  let list4MapIndexExn = (~f, ~index, l) =>
+    switch (l) {
+    | [a, b, c, d] =>
```

Comment:
Could be a tuple? Not sure if is worth it. 😄

Reply:
It was a tuple at first but then I had to write a map function and it became more verbose: https://github.com/briskml/brisk-reconciler/pull/22/files/b673a6863092aea4718bd8f55d869c54e04d58e0#diff-3192c885d98e8499edfef30e642aaf4dR118